tools/Config.mk

```diff
@@ -578,8 +578,8 @@ $(1)_$(2):
 endef
 
 export DEFINE_PREFIX := $(subst X,,${shell $(DEFINE) "$(CC)" "X"})
```

Comment:
why change this line

Reply:
it didn't change

Reply:
sorry, I mean why not change this line too